### PR TITLE
Removes the last of the red cross

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Removes the last of the red cross from sprites and typepaths in this game. I am sure youre all wondering, "gee affected, whats so bad about a little red cross sprite?". Well I'll tell you. **It violates the fucking geneva convention**

Take a read here, https://www.pcgamer.com/how-the-prison-architect-developers-broke-the-geneva-conventions/

PR which did part of this: https://github.com/ParadiseSS13/Paradise/pull/8494

## Why It's Good For The Game
Warcrimes are bad

## Images of changes
Nothing major, and the diffbots will handle it. If requested, I will take actual photos

## Changelog
:cl: AffectedArc07
del: Removes red cross insignia from the game, for reasons
/:cl:

### CODE REVIEW

File: _maps/map_files/shuttles/admin_hospital.dmm [hunk 1]
```diff
@@ -1,1673 +1,1669 @@
...
-aW
-aa
-aa
-an
-aM
-aM
-aW
-aa
-"}
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (AffectedArc07) [inline _maps/map_files/shuttles/admin_hospital.dmm:1]: Oh shit line endings

**Comment by Regular-Joe-SF** [comment]
Technically, the authorization thing with the usage of the red cross, is about limiting its usage on either unarmed medical personnel, or civilian and military medical facilities, and the authorization depends on whether they actually are what they are marked to be; one may use the symbol if that is the case. Then attacking anything marked with that is considered a war crime, unless done on retalition. So is using the symbol on something that has an aggression capability.

So well, a medbay could be symboled with a red cross in a game, as well as an ambulance, but well... who spaceman or syndicate respects the Space Geneva convention, so that a worry over people getting used to bomb the red cross, so eating up the symbol, is indeed a real one!

**Comment by Spacemanspark** [comment]
What is the world coming to

**Comment by AffectedArc07** [comment]
@Spacemanspark shite, thats what

**Comment by Gatchapod** [comment]
> 
> 
> Technically, the authorization thing with the usage of the red cross, is about limiting its usage on either unarmed medical personnel, or civilian and military medical facilities, and the authorization depends on whether the things actually are what they are marked to be; one may use the symbol if that is the case. Then attacking anything marked with that, as a part of an organization, is considered a war crime, unless done on retalition. So is using the symbol on something that has an aggression capability.
> 
> So well, a medbay could be symboled with a red cross in a game, as well as an ambulance, but well... who spaceman or syndicate respects the Space Geneva convention, so that a worry over people getting used to bomb the red cross, so eating up the symbol, is indeed a real one!

In this case, I do not believe marking our medbay with the Red Cross symbol would be appropriate.
There are universal symbols of medicine, like rod of Asclepius, which carry a lot less implications than the Red Cross.
Putting the cross on medbay would imply it's not strictly an NT facility. RC will respect the laws of the territory its stationed in, but their humanitarian approach will complicate things significantly if your laws include death penalty.
I am fairly certain that an RC doctor would be under no obligation to cease resuscitation of a dangerous criminal based on a security officer's order. In fact, RC would likely only deploy if their doctors were guaranteed protection in such a case, and ability to carry out their duties undisturbed. They'd likely be outright obliged to disrespect any executions carried out without a proper trial, including KoS orders or self-evident crimes and verbal execution approvals.

I know, I know. This is just a silly spessman simulation game and no one actually ever associated those crosses with **the** Red Cross. I am being stupid here.
One way or another, we probably won't have the symbol after this PR at all.

**Comment by Regular-Joe-SF** [comment]
Heh yeah Gatchapod, an interesting nerdy topic to think about (anyway the actual PR is totally a good idea so uh, off topic, honk.) The conventions also allow the usage of the cross for stuff not related an authorized IRC organization, if it used on the mentioned terms, not though by an individual or such, but a state organization/uniformed group. But a corporation executing legal force in space, would that be an ”uniformed” organization the GC allows the symbol and on whose the GC also generally deals with? Future problems, I guess!

**Comment by AffectedArc07** [comment]
The latest commit removes the fluff sprite with the ID of `asusrose`. This isnt registered to an item ingame, and is too difficult to resprite into being green.

**Pull request merged**